tests/foreman/api/test_eol_banner.py

```diff
@@ -0,0 +1,49 @@
+"""Test module for EOL tracking
+
+:Requirement: Dashboard
+
+:CaseAutomation: Automated
+
+:CaseComponent: Dashboard
+
+:Team: Endeavour
+
+:CaseImportance: High
+"""
+
+from datetime import datetime
+
+import pytest
+import requests
+import yaml
+
+from robottelo.config import settings
+from robottelo.constants import LIFECYCLE_METADATA_FILE
+
+
+@pytest.mark.tier2
+def test_positive_check_eol_date(target_sat):
+    """Check if the EOL date for the satellite version
+
+    :id: 1c2f0d19-a357-4461-9ace-edb468f9ca5c
+
+    :expectedresults: EOL date from satellite-lifecycle package is accurate
+    """
+    current_version = '.'.join(target_sat.version.split('.')[0:2])
+    output = yaml.load(target_sat.execute(rf'cat {LIFECYCLE_METADATA_FILE}').stdout, yaml.Loader)
+    eol_datetime = datetime.strptime(output['releases'][current_version]['end_of_life'], '%Y-%m-%d')
+    result = requests.get(settings.subscription.lifecycle_api_url, verify=False)
+    if result.status_code != 200:
+        raise requests.HTTPError(f'{settings.subscription.lifecycle_api_url} is not accessible')
```

Comment:
```suggestion
    result.raise_for_status()
```
https://docs.python-requests.org/en/latest/api/#requests.Response.raise_for_status